include/uv-os390.h

```diff
@@ -26,5 +26,9 @@
 
 #define UV_PLATFORM_LOOP_FIELDS                                               \
   void* ep;                                                                   \
+  int fs_event_msg_queue;                                                     \
+
+#define UV_PLATFORM_FS_EVENT_FIELDS                                           \
+  char rfis_rftok[8];                                                         \
```

Comment:
Yes, it's semver-major.  @jBarz If you want to see this land in v1.x, you should devise something that keeps the struct sizes the same.